mne/viz/evoked.py

```diff
@@ -1028,6 +1029,11 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
     if picks is not None:
         pick_names = [evoked.info['ch_names'][pick] for pick in picks]
         evoked.pick_channels(pick_names)
+    else:  # only pick channels that are plotted
+        picks = _pick_data_channels(evoked.info, exclude=[])
+        pick_names = [evoked.info['ch_names'][pick] for pick in picks]
+    evoked.pick_channels(pick_names)
```

Comment:
Yup, missed during debug ... Did you check if I did the projection thing correctly?